README.md

```diff
@@ -1 +1,72 @@
-# cytoscape.js-klay
\ No newline at end of file
+cytoscape-klayjs
+================================================================================
+
+
+## Description
+
+KLayJS for Cytoscape
+
+
+## Dependencies
+
+ * Cytoscape.js ^x.y.z
+ * <List your dependencies here please>
+
+
+## Usage instructions
+
+Download the library:
+ * via npm: `npm install cytoscape-klayjs`,
+ * via bower: `bower install cytoscape-klayjs`, or
+ * via direct download in the repository (probably from a tag).
+
+`require()` the library as appropriate for your project:
+
+CommonJS:
+```js
+var cytoscape = require('cytoscape');
+var klayjs = require('cytoscape-klayjs');
+
+klayjs( cytoscape ); // register extension
+```
+
+AMD:
+```js
+require(['cytoscape', 'cytoscape-klayjs'], function( cytoscape, klayjs ){
+  klayjs( cytoscape ); // register extension
+});
+```
+
+Plain HTML/JS has the extension registered for you automatically, because no `require()` is needed.
+
+
+## API
+
+Please briefly describe your API here:
```

Comment:
An example of running the layout should be here

Example: https://github.com/cytoscape/cytoscape.js-cola